Add a description about new data mapper code action (#1136)
ballerina-platform/ballerina-dev-website: swan-lake/learn/tools-ides/setting-up-visual-studio-code/language-intelligence.md

- Add missing imports 
- Create variable definitions
- Create an undefined function

> - Map record types automatically

Reviewer: ```suggestion
- Map the record types automatically
```

Reviewer 2: It is fine to say "Map record types" here as we aren't specifying a particular record type but referring to record types in general.

> For example, you can add documentation for a function as shown below.

Reviewer 2: ```suggestion
For another example, you can add documentation for a function as shown below.
```
Let's move this to place it after the data mapping code action content.

 ![Code actions](/swan-lake/learn/images/code-actions.gif)

> You can use automatic data mapping code action as shown below.

Reviewer 2: ```suggestion
For example, you can use the automatic data mapping code action as shown below.
```

>  > **Note**: By default, the Automatic data mapping code action is disabled. In order to enable it, go to **Settings** and Select **Ballerina** from **Extensions**. There you can check the box in front of **Data Mapper: Enabled**. 

Reviewer 2: ```suggestion
 > **Note**: By default, the automatic data mapping code action is disabled. To enable it, click **Settings**, select **Ballerina** from **Extensions**, and select the **Data Mapper: Enabled** checkbox. (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.)
```

>  > **Note**: By default, the automatic data mapping code action is disabled. To enable it, click **Settings**, select **Ballerina** from **Extensions**, and select the **Data Mapper: Enabled** checkbox. (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.)

Reviewer: Why do we need to repeat the sentence ` (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.` here and as a `Tip` below.

Reviewer 2: The review comment was to remove the tip and add this in brackets.

Author: With the new commit the repeated sentence was removed

 ![Code actions](/swan-lake/learn/images/data-mapper-code-action.gif)

>  > **Tip**: If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.

Reviewer 2: ```suggestion
 > **Tip**: If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.
```
This tip can be removed.

Author: Removed this tip

- Add missing imports 
- Create variable definitions
- Create an undefined function
- Map the record types automatically

For example, you can use the automatic data mapping code action as shown below.

>  > **Note**: By default, the automatic data mapping code action is disabled. To enable it, click **Settings**, select **Ballerina** from **Extensions**, and select the **Data Mapper: Enabled** checkbox. (If you are using MacOS, you can go to **Settings** by pressing `Cmd` + `,`.)

Reviewer 2: ```suggestion
 > **Note**: By default, the automatic data mapping code action is disabled. To enable it, click **Settings**, select **Ballerina** from **Extensions**, and select the **Data Mapper: Enabled** checkbox. (If you are using macOS, you can go to **Settings** by pressing `Cmd` + `,`.)
```
